website/docs/how-to/how-to-use-the-admin-api.md

```diff
@@ -2,36 +2,34 @@
 title: How to use the Admin API
 ---
 
-This guide explains the steps required to getting access to and using the Admin API.
+This guide explains the steps required to using the Admin API.
 
-## Step 1: Create API token {#step-1-create-api-token}
+## Create API token
 
-You'll need either an [admin token](../reference/api-tokens-and-client-keys.mdx#admin-tokens) or a [personal access token](../reference/api-tokens-and-client-keys.mdx#personal-access-tokens) for this to work. To create one, follow the steps in the [_how to create API tokens_](../how-to/how-to-create-api-tokens.mdx) guide or the [_how to create personal access tokens_](./how-to-create-personal-access-tokens.mdx) guide, respectively.
+First, you'll need to create a [personal access token](/reference/api-tokens-and-client-keys.mdx#personal-access-tokens).
 
 Please note that it may take up to 60 seconds for the new key to propagate to all Unleash instances due to eager caching.
 
 :::note
 
-If you need an API token to use in a client SDK you should create a "client token" as these have fewer access rights.
+If you need an API token to use in a client SDK you should create a client token instead, as these have fewer access rights.
 
 :::
 
-## Step 2: Use Admin API {#step-2-use-admin-api}
+## Use Admin API
 
-Now that you have an access token with admin privileges we can use that to perform changes in our Unleash instance.
+Now that you have an access token with admin privileges, you can use it to make changes in your Unleash instance.
 
-In the example below we will use the [Unleash Admin API](/reference/api/legacy/unleash/admin/features.md) to enable the “Demo” feature flag using curl.
+In the example below we will use the [Unleash Admin API](/reference/api/legacy/unleash/admin/features.md) to enable the `Demo` feature flag using curl.
 
 ```sh
 curl -X POST -H "Content-Type: application/json" \
-     -H "Authorization: some-token" \
+     -H "Authorization: <your-token>" \
      https://app.unleash-hosted.com/demo/api/admin/features/Demo/toggle/on
```

Comment:
I know this is unrelated to the PR but I'm pretty sure this endpoint is wrong. I believe it should be something like: `https://app.unleash-hosted.com/demo/api/admin/projects/<your-project-id>/features/<your-flag-name>/environments/<your-environment>/on`.

Reply:
nice catch, thanks!